float64 alt_ref_w
---
bool  update_status